csunplugged/static/scss/website.scss

```diff
@@ -20,3 +20,6 @@ h1, h2, h3 {
     color: #888;
   }
 }
+input[type="text"].long-text-field {
+  width: 20em;
+}
```

Comment:
missing newline (from my commit), you might as well fix before merging into develop